app/models/season/phase_switcher.rb

```diff
@@ -4,6 +4,16 @@ def self.season
     Season.current
   end
 
+  def self.switcher(phase)
+    @phases = {
+        'Proposals' => :fake_proposals_phase,
```

Comment:
Would it make sense to move this traffic control plus the whitelisting back to the controller?